docs/src/components/pricing/WhatToExpect.tsx

```diff
@@ -78,11 +78,14 @@ export default function WhatToExpect() {
               </Typography>
             </Box>
             <Typography variant="body2" color="text.secondary">
-              With your purchase you receive support, and access to new versions, for the duration of
-              your subscription. You can learn more about{' '}
-              <Link href="https://mui.com/x/advanced-components/#support">support in the docs</Link>
-              . Note that we roll bug fixes, performance enhancements, and other improvements into
-              new release, instead of patching older versions.
+              With your purchase you receive support, and access to new versions, for the duration
+              of your subscription. You can{' '}
+              <Link href="https://mui.com/x/advanced-components/#support">
+                learn more about support
+              </Link>{' '}
+              in the docs. Note that, except for critical issues, e.g. security, we release bug
```

Comment:
```suggestion
              in the docs. Note that, except for critical issues such as those impacting security, we release bug
```

Reply:
We don't have enough space for this suggestion, or we would need to find a way to balance it https://deploy-preview-32811--material-ui.netlify.app/pricing/

Reply:
Thanks for the feedback. I hadn't looked at it in context, but do now see the problem. In this case feel free to disregard the review.